### PR TITLE
feat: Add roadmap

### PR DESCRIPTION
**Pull request by gaocegege**
PTAL

Especially 2020.09 release.

/assign @ddysher @codeflitting @simon-cj 

Signed-off-by: Ce Gao <ce.gao@outlook.com>

### CODE REVIEW

**Comment by gaocegege** [comment]
@bbbmj I am using non-caicloud mail address to commit, but got cla failure here. How can we fix it? Since we have non-caicloud members, we should support signing CLA instead of using caicloud mail address.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,69 @@
...
+<a href="https://join.slack.com/t/kleveross/shared_invite/zt-g0eoiyq9-9OwiI7c__oV79bh_94MyTw">
+    <img src="https://cdn.brandfolder.io/5H442O3W/as/pl546j-7le8zk-5guop3/Slack_RGB.png" alt="Slack" height =30px/></a>
+
 Cloud Native ML/DL Platform
+
+## Roadmap
+
+### 2020.09
+
+The 2020.09 release contains klever-model-registry and ormb.
```
Comment (deyuanix) [inline README.md]: nit: we can add links to the repos here.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,69 @@
...
+
+klever-model-registry contains one long-running backend server, one Kubernetes CRD operator, one dashboard.
+
+klever-model-registry features:
+
+- Manage ML models
+- Keep track of ML models' hyperparameters and so on to help decision makers
+- Convert models between different formats
+- Extract signatures for the given models
+- Serve the simple models using Triton Inference Server
```
Comment (deyuanix) [inline README.md]: what is `simple model`?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,69 @@
...
+- Support uploading models from the dashboard
+- Get job logs and events
```
Comment (deyuanix) [inline README.md]: nit: clarify what is the `job` here.

**Comment by codeflitting** [comment]
LGTM

**Review by deyuanix — COMMENTED** [review]
Most of the information here are feature-wise, do we need something like release process?

File: README.md (hunk 2 above)
Comment (gaocegege) [reply README.md]: Single models, we do not support inference graphs.

File: README.md (hunk 3 above)
Comment (gaocegege) [reply README.md]: SGTM

**Comment by gaocegege** [comment]
> do we need something like the release process?

I think we need it. Will add it soon.

**Comment by bbbmj** [comment]
> @bbbmj I am using non-caicloud mail address to commit, but got cla failure here. How can we fix it? Since we have non-caicloud members, we should support signing CLA instead of using caicloud mail address.

We'll remove caicloud-cla first.

**Comment by gaocegege** [comment]
Ref #2

**Comment by gaocegege** [comment]
@ddysher 

Comments are addressed, please take a look.
Thanks for your time :smile:

**Comment by gaocegege** [comment]
I am adding owners in the PR and merge it manually.

**Pull request merged**